locale/en/download/releases.md

```diff
@@ -3,4 +3,5 @@ layout: download-releases.hbs
 title: Previous Releases
 iojs:
   intro: "Releases 1.x through 3.x were called \"io.js\" as they were part of the io.js fork. As of Node.js 4.0.0 the former release lines of io.js converged with Node.js 0.12.x into unified Node.js releases."
+modules: "\"Modules\" refers to the ABI (application binary interface) version number of Node.js, used to determine which versions of Node.js compiled C++ add-on binaries can be loaded in to without needing to be re-compiled. This version number is also referred to as NODE_MODULE_VERSION."
```

Comment:
@Fishrock123 the period is in the diff, not in the screenshot sorry. Heres what it looks like wrapped with a `<code>`:

<img width="302" alt="screen shot 2015-09-19 at 10 09 59 am" src="https://cloud.githubusercontent.com/assets/39759/9977305/b9659c98-5eb6-11e5-819d-9eab5621c07c.png">

And without (the current version in the branch now):

<img width="313" alt="screen shot 2015-09-19 at 10 10 29 am" src="https://cloud.githubusercontent.com/assets/39759/9977307/c393c10e-5eb6-11e5-8af4-156028653fb3.png">